src/jsx/components/input_info.jsx

```diff
@@ -0,0 +1,31 @@
+var React = require('react'),
+    ReactDOM = require('react-dom');
+
+var InputInfo = React.createClass({
```

Comment:
Maybe add "getDefaultProps" so that we know which input_data object attributes should be available.